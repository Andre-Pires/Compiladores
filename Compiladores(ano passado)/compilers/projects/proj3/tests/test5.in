t1 = a
t2 = b
t3 = t1 + t2
a = t3
