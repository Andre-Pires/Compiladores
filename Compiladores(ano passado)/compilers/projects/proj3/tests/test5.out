 Number of Registers: 4
	r2 = ARP + 4
	r2 = *r2
	r2 = r2
	r1 = ARP + 8
	r1 = *r1
	r1 = r1
	r0 = r2 + r1
	r0 = r0
	r3 = ARP + 4
	*r3 = r0
 Number Load Operations: 2
 Number Store Operations: 1
